docs/testing/web/colour-contrast.md

```diff
@@ -1,6 +1,6 @@
 # Colour contrast
 
-We must always meet the WCAG 2.0 AAA standard for colour contrast.
+We must always meet the WCAG 2.1 AA standard for colour contrast.
```

Comment:
```suggestion
We must strive to meet WCAG 2.1 AAA and always meet the WCAG 2.1 AA standard for colour contrast.
```
?

Reply:
Hmm. I feel like having a single clear rule is simpler to follow than having a minimum standard and then also a stretch goal. We will just end up hitting the minimum in this case. AAA is hard to achieve even when you’re aiming for it!